bokeh/protocol/message.py

```diff
@@ -107,8 +106,9 @@ class Header(_Header, total=False):
     reqid: ID
     num_buffers: int
 
-class BufferHeader(TypedDict):
-    id: ID
+# class BufferHeader(TypedDict):
+#     id: ID
+BufferHeader = str
```

Comment:
The TypedDict does not reflect that actual usage anywhere. It seems like a nice idea but commenting until the other corresponding necessary changes can be made.

Reply:
This module gave me a hard time when initially trying to type it, so I can't vouch for those types.